src/BloomBrowserUI/bookEdit/js/textOverPicture.ts

```diff
@@ -368,7 +386,19 @@ export class TextOverPictureManager {
                 );
             } else {
                 // Not currently dragging
-                if (Comical.getBubbleHit(container, ev.offsetX, ev.offsetY)) {
+                const targetElement = ev.target as HTMLElement;
+                const targetBounds = targetElement.getBoundingClientRect();
+
+                // These coordinates need to be relative to the canvas (which is the same as relative to the image container).
+                const [targetX, targetY] = this.getCoordinatesRelativeTo(
+                    ev.offsetX,
+                    ev.offsetY,
+                    targetBounds,
+                    containerBounds,
+                    styleInfo
+                );
+
+                if (Comical.getBubbleHit(container, targetX, targetY)) {
                     // But could be dragging a bubble, so make the mouse indicate that
```

Comment:
But the logic above (and comment) says we are NOT dragging. Or maybe the comment should be, "But we're over a bubble that could be dragged..."